packages/main/src/webComponents/Input/index.tsx

```diff
@@ -106,14 +106,10 @@ export interface InputPropTypes extends InputAttributes, Omit<CommonProps, 'onCh
    *
    * Example:
    *
-   * <pre>
-   *   <code>
-   *    &lt;Input showSuggestions><br />
-   *    &nbsp;&nbsp;&lt;SuggestionItem text="Item #1" /><br />
-   *    &nbsp;&nbsp;&lt;SuggestionItem text="Item #2" /><br />
-   *    &lt;/Input>
-   *  </code>
-   * </pre>
+   * <Input show-suggestions>
+   *     <SuggestionItem text="Item #1"></SuggestionItem>
+   *     <SuggestionItem text="Item #2"></SuggestionItem>
+   * </Input>
```

Comment:
I think we need to encode the HTML tags here, otherwise storybook will attempt to render the input?

Reply:
Yup, that was an oversight after running the script. Thanks for noticing!